components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -4,48 +4,13 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FunctionComponent } from "react";
-import { Redirect } from "react-router";
-import Header from "../components/Header";
-import { SpinnerLoader } from "../components/Loader";
-import { useCurrentOrg } from "../data/organizations/orgs-query";
 import { GitIntegrations } from "./git-integrations/GitIntegrations";
 import { OrgSettingsPage } from "./OrgSettingsPage";
 
-export default function GitAuth() {
+export default function GitAuthPage() {
     return (
-        <OrgSettingsPageWrapper>
+        <OrgSettingsPage title="Git Auth" subtitle="Configure Git Auth for GitLab or Github.">
```

Comment:
Wrapper component no longer needed, as the loading state is handled in `<OrgSettingsPage/>` now.

Reply:
**issue**: I wanted to mention this in https://github.com/gitpod-io/gitpod/pull/16850, but the goal of the settings page was to keep the _Organizations Settings_ in the header around for all pages under settings, and rely on the left sidebar for navigation and less duplication of labels. Cc @AlexTugarev 
 
| Example of labels repeating |
|-|
| <img width="1163" alt="Frame 1303" src="https://user-images.githubusercontent.com/120486/225615723-bd927436-ec9f-4bfc-8dde-48a56b6c4331.png"> |

Reply:
Oh, that's my mistake then, I understood some of that differently. I can change this back if that's what we want in a followup pretty easily. It is kinda awkward coming up with 3 versions of the page title that are slightly different, so this would remove that 😄